client/src/sass/application.scss

```diff
@@ -331,3 +331,38 @@ table {
     }
   }
 }
+
+// overflow-databale responsive rules
+
+@media screen and (max-width: #{map-get($grid-breakpoints, xxl)}) {
+  .main-col {
+    &.expanded {
+      @include overflow-datatable(map-get($grid-breakpoints, lg), $expanded-horizontal-margins/2);
+    }
+
+    &:not(.expanded) {
+      @include overflow-datatable(map-get($grid-breakpoints, lg), $not-expanded-horizontal-margins + $menu-width/2);
+    }
+  }
+}
+
+@media screen and (max-width: #{map-get($grid-breakpoints, lg)}) {
+  .main-col {
+    &.expanded {
+      @include overflow-datatable(map-get($grid-breakpoints, lg), $expanded-horizontal-margins/3);
+    }
+
+    &:not(.expanded) {
+      // TODO: $horizontal-margins might change to $expanded-horizontal-margins/3 + $menu-width/2
+      @include overflow-datatable(map-get($grid-breakpoints, lg), $not-expanded-horizontal-margins + $menu-width/2);
+    }
```

Comment:
Required the PR https://github.com/Chocobozzz/PeerTube/pull/2698 to change the `$horizontal-margins` and compact this block :

Reply:
I merged the aforementioned PR.

Reply:
It got merged :slightly_smiling_face: